css/cssom/caretRangeFromPoint-textarea-transform.tentative.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<meta charset="utf-8" />
+<title>document.caretRangeFromPoint()</title>
+<link rel="help" href="https://github.com/w3c/csswg-drafts/pull/12362" />
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+  textarea {
+    display: block;
+    translate: -50%;
+  }
+</style>
+<textarea id="textarea" style="translate: -50% -50%">
```

Comment:
Do you need the inline `translate` here also? You've got that in the `<style>` above.